_posts/2013-01-15-never-use-auto-play.md

```diff
@@ -7,42 +7,48 @@ categories:
   - Quick Tips
 ---
 
-**Summary\:** don't auto-play and it will please *all* of your users.
+Not auto-playing your media will make everyone happy.
 
 ## Sound
 
-People using screen-readers navigate by listening, so any sound playing when the page loads will interfere immensely.
+People using screen-readers navigate by listening, so any sound playing when the page loads will interfere immensely. It will also be frustrating to any person that may be listening to music or is in a quiet area with their sound on high.
 
-In [a note on audio control](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/visual-audio-contrast-dis-audio.html), WCAG 2.0 suggests:
+In [a note on audio control](https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/visual-audio-contrast-dis-audio.html), the Web Content Accessibility Guidelines (WCAG) 2.0 suggests:
```

Comment:
let's update 2.0 to 2.1 or remove the version designation to not seem out of date?

Reply:
Good catch. Done.